fixtures/docstring-proc-macro/src/lib.rs

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::sync::Arc;
+
+/// <docstring-enum>
+#[derive(uniffi::Enum)]
+enum EnumTest {
+    /// <docstring-enum-variant>
+    One,
+    /// <docstring-enum-variant-2>
+    Two,
+}
+
+/// <docstring-associated-enum>
+#[derive(uniffi::Enum)]
+pub enum AssociatedEnumTest {
+    /// <docstring-associated-enum-variant>
+    Test {
+        /// <docstring-variant-field>
+        code: i16,
+    },
+    /// <docstring-associated-enum-variant-2>
+    Test2 { code: i16 },
+}
+
+/// <docstring-error>
+#[derive(uniffi::Error, Debug, thiserror::Error)]
+enum ErrorTest {
```

Comment:
If you add `#[uniffi(flat_error)]` here, the `message` field is exposed and the test file would be the same as for the `docstring` fixture.

Reply:
Nice, I missed that. Read the documentation too fast...